src/main/resources/db/changelog/db.changelog-insert.yaml

```diff
@@ -79,4 +69,28 @@ databaseChangeLog:
                   value: 1
               - column:
                   name: product_id_id
-                  value: 1
\ No newline at end of file
+                  value: 1
+        - insert:
+            tableName: comment
+            columns:
+              - column:
+                  name: user_id_id
```

Comment:
as you named you manytoone relationships as userId now you shoud name it as user_id_id in db